src/docs/MLOS/MLOSUserGuide.adoc

```diff
@@ -591,6 +594,8 @@ The returned values of your final tasks will be automatically printed.
 Finally, to have the hand on more parameters and features, the user should use ActiveEon Studio portals.
 The main ones are the _Resource Manager_, the _Scheduling Portal_ and the _Workflow Automation_.
 
+The example below shows how the user can directly monitor his submitted job's execution in the scheduling portal:
```

Comment:
The example below shows how the user can directly monitor the execution of submitted jobs in the scheduling portal:

Reply:
I prefer keeping the original sentence. It puts more emphasis on which execution and which job we are talking about.

Reply:
`his submitted job's execution ` -> Too long.  Prop: `can directly monitor the submitted jobs during their execution phase in the ...`